src/main/scala/scorex/core/NodeViewHolder.scala

```diff
@@ -294,8 +304,8 @@ trait NodeViewHolder[TX <: Transaction, PMOD <: PersistentNodeViewModifier]
 
               case Failure(e) =>
                 log.warn(s"Can`t apply persistent modifier (id: ${pmod.encodedId}, contents: $pmod) to minimal state", e)
+                // not publishing SemanticallyFailedModification as this is an internal error
                 updateNodeView(updatedHistory = Some(newHistory))
-                context.system.eventStream.publish(SemanticallyFailedModification(pmod, e))
```

Comment:
@kushti I removed this because this Failure can only happen due to some unexpected internal error, we already fire it in the `applyState` method